salvo/src/lib/benchmark/base_benchmark.py

```diff
@@ -0,0 +1,68 @@
+"""
+Base Benchmark object module that contains
+options common to all execution methods
+"""
+import os
+import logging
+
+import src.lib.docker_image as docker_image
+import src.lib.docker_volume as docker_volume
+
+log = logging.getLogger(__name__)
+"""
+Base Benchmark class with common functions for all invocations
+"""
+
+
+def get_docker_volumes(output_dir, test_dir=None):
```

Comment:
A question on privacy - are all of these functions and method meant to be public? (will they be used from outside of this package?

If some of them aren't we should prefer to make them private by prepending an underscore to their name. Private methods make it easier to change code around in the future.

https://github.com/google/styleguide/blob/gh-pages/pyguide.md#3162-naming-conventions

Reply:
Yep, I can take a look and make the functions private where applicable